CHANGELOG.md

```diff
@@ -59,6 +59,8 @@ v0.33.0-rc.0 (2023-04-20)
 ### Deprecations
 
 - [Dynamic Configuration](https://grafana.com/docs/agent/latest/cookbook/dynamic-configuration/) will be removed in v0.34. Grafana Agent Flow supersedes this functionality. (@mattdurham)
+- Remove deprecated symbolic links to to `/bin/agent*` in Docker containers,
+  as planned in v0.31. (@tpaschalis)
```

Comment:
This is a breaking change, not a deprecation (it was previously noted as a deprecation)